src/app/utils/elementCreator.js

```diff
@@ -0,0 +1,12 @@
+import elementInjector from './elementInjector';
+
+const elementCreator = (type, attributes, ...children) => {
```

Comment:
Here you use 
`const f = () => {}`
and in elementInjector
`function (){}`. Is there a reason for it?